app.py

```diff
@@ -243,10 +256,11 @@ async def generate(self, prompt: Union[Prompt, TextPrompt]):
                 except Exception as e:
                     print(e, flush=True)
 
+        validatorItems = self.available_validators.items()
         hotkeys = [
             hotkey
-            for hotkey, log in self.available_validators.items()
-            if log["is_active"]
+            for hotkey, log in validatorItems
+            # if log["is_active"]
```

Comment:
why we don't check is_active?

Reply:
I commented it to test on my local environment. Will uncomment it back.